src/components/Tokens/TokensBanner.tsx

```diff
@@ -0,0 +1,86 @@
+import { PageName } from 'components/AmplitudeAnalytics/constants'
+import { useAtom } from 'jotai'
+import { getCurrentPageFromLocation } from 'pages/App'
+import { useEffect, useState } from 'react'
+import { X } from 'react-feather'
+import { useLocation } from 'react-router-dom'
+import styled, { useTheme } from 'styled-components/macro'
+import { opacify } from 'theme/utils'
+
+import tokensPromoDark from '../../assets/images/tokensPromoDark.png'
+import tokensPromoLight from '../../assets/images/tokensPromoLight.png'
+import { tokensBannerDismissedAtom } from './state'
+
+const PopupContainer = styled.div<{ show: boolean }>`
+  position: absolute;
+  display: ${({ show }) => (show ? 'flex' : 'none')};
+  flex-direction: column;
+  padding: 12px 16px 12px 20px;
+  gap: 8px;
+  bottom: 48px;
+  right: 16px;
+  width: 320px;
+  height: 88px;
+  z-index: 5;
+  background-color: ${({ theme }) => (theme.darkMode ? theme.backgroundScrim : opacify(60, '#FDF0F8'))};
```

Comment:
why the specific case for lightMode here?

Reply:
design had dark/light specific designs for this component in particular (noted in figma and the jira ticket)